### PR TITLE
Add role badges to invocations

### PR DESCRIPTION
**Pull request by bduffany**
History cards:

image

Invocation overview:

image

---

**Version bump**: Patch 

**Related issues**: Fixes https://github.com/buildbuddy-io/buildbuddy-internal/issues/619

### CODE REVIEW

File: enterprise/app/history/history_invocation_card.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { CSSProperties } from "react";
```
Comment (siggisim) [inline enterprise/app/history/history_invocation_card.tsx]: is this import needed?

File: app/root/root.css [hunk 2]
```diff
@@ -824,6 +824,25 @@ code .comment {
   margin-left: 0;
 }
 
+.titles .role-badge {
+  margin-right: 8px;
+}
+
+.role-badge {
+  border-radius: 12px;
+  padding: 0 8px;
```
Comment (siggisim) [inline app/root/root.css:833]: how does this look with like 2px of vertical padding?

File: enterprise/app/history/history_invocation_card.tsx [hunk 3]
```diff
@@ -214,7 +217,7 @@ export default class HistoryInvocationCardComponent extends React.Component {
                   {this.props.invocation.host}
                 </div>
               )}
-              {this.props.invocation.command && (
+              {this.props.invocation.command && this.props.invocation.command != "workflow run" && (
```
Comment (siggisim) [inline enterprise/app/history/history_invocation_card.tsx:220]: can we keep the tool icon?

**Review by siggisim — APPROVED** [review]

File: enterprise/app/history/history_invocation_card.tsx (hunk 1 above)
Comment (siggisim) [reply enterprise/app/history/history_invocation_card.tsx]: looks like you already removed it

File: app/root/root.css (hunk 2 above)
Comment (bduffany) [reply app/root/root.css]: Looks good: 
image

File: enterprise/app/history/history_invocation_card.tsx (hunk 3 above)
Comment (bduffany) [reply enterprise/app/history/history_invocation_card.tsx]: woops, done. This made more sense when rendering the role as one of the details (it looked awkward having 2 details that both said "workflow")

**Pull request merged**